code/modules/mob/living/carbon/human/species/station/teshari.dm

```diff
@@ -58,7 +58,7 @@
 
 	ambiguous_genders = TRUE
 
-	spawn_flags = SPECIES_CAN_JOIN | SPECIES_IS_WHITELISTED
+	spawn_flags = SPECIES_CAN_JOIN | SPECIES_IS_WHITELISTED	| SPECIES_NO_POSIBRAIN
```

Comment:
This is already overridden in teshari_vr so it won't affect us, which is probably a good thing.